pkg/services/publicdashboards/api/api.go

```diff
@@ -131,6 +136,17 @@ func (api *Api) SavePublicDashboard(c *models.ReqContext) response.Response {
 	return response.JSON(http.StatusOK, pubdash)
 }
 
+// Delete a public dashboard
+// DELETE /api/dashboards/uid/:dashboardUid/public-dashboards/:uid
+func (api *Api) DeletePublicDashboard(c *models.ReqContext) response.Response {
+	err := api.PublicDashboardService.Delete(c.Req.Context(), c.SignedInUser.OrgID, web.Params(c.Req)[":dashboardUid"], web.Params(c.Req)[":uid"])
```

Comment:
```suggestion
	err := api.PublicDashboardService.Delete(c.Req.Context(), c.OrgID, web.Params(c.Req)[":dashboardUid"], web.Params(c.Req)[":uid"])
```

Reply:
This change isn't strictly necessary. It's actually the same variable, however, we're using c.OrgID everywhere else.